Scripts/travis.sh

```diff
@@ -108,7 +108,9 @@ elif [ "${TYPE}" == "EXAMPLE_PODS_SWIFT" ]; then
   cd gem/
   bundle install
   rake install
-  cd ..
+  cd ../Demo/EarlGreyExample
+  pod update
```

Comment:
`pod update` should always happen after `pod install`, right?

I am curious to see the travis log, it sounds alarming to me that every run in Travis is different, at lease in a way they cache stuff from others...